### PR TITLE
fix issues in sample app while regression of 3.0.0

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Fix some issues that is reported in Android regression 3.0.0

## Links
- MINI-2941
- MINI-2942

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt [hunk 1]
```diff
@@ -89,11 +92,32 @@ class MiniAppInputActivity : MenuBaseActivity(), PreloadMiniAppWindow.PreloadMin
...
         is InputDisplay.Url -> MiniAppDisplayActivity.startUrl(this, display.input.trim())
         is InputDisplay.None -> {}
     }
 
+    private fun initiatePreloadScreen(miniAppId: String) {
+        val viewModel: MiniAppInputViewModel = ViewModelProvider.NewInstanceFactory().create(MiniAppInputViewModel::class.java)
+                .apply {
+                    // observe version id
+                    miniAppVersionId.observe(this@MiniAppInputActivity, Observer {
+                        preloadMiniAppWindow.initiate(
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt:107]: I can pass a valid versionId from input after observing it in order to fix MINI-2941.

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt [hunk 2]
```diff
@@ -89,11 +92,32 @@ class MiniAppInputActivity : MenuBaseActivity(), PreloadMiniAppWindow.PreloadMin
     }
 
     private fun displayMiniApp() = when(display) {
-        is InputDisplay.AppId -> preloadMiniAppWindow.initiate(null, display.input.trim(), this)
+        is InputDisplay.AppId -> {
+            initiatePreloadScreen(display.input.trim())
+        }
```
Comment (minh-rakuten) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt:97]: You can remove the bracket here to have a single line.

**Review by minh-rakuten — APPROVED** [review]
LGTM

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt (hunk 2 above)
Comment (khairul-alam-licon) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt]: Yeah, removed the bracket.

**Pull request merged**